src/agent/v8/legacy-debugapi.ts

```diff
@@ -320,6 +307,19 @@ export class V8DebugApi implements debugapi.DebugApi {
       column += debugapi.MODULE_WRAP_PREFIX_LENGTH - 1;
     }
 
+    // TODO: Address the case where `fileStats[matchingScript]` is `null`.
+    if (line >= (this.fileStats[matchingScript] as FileStats).lines) {
+      return utils.setErrorStatusAndCallback(
+          cb, breakpoint, StatusMessage.BREAKPOINT_SOURCE_LOCATION,
+          utils.messages.INVALID_LINE_NUMBER + matchingScript + ':' +
+              (breakpoint.location as stackdriver.SourceLocation).line +
```

Comment:
same as the other. should this be `line`?

Reply:
Yep.  Fixed.